### ISSUE TITLE
Little message redirecting folks looking for datacap?

### ISSUE THREAD

**Issue by dannyob** [data-preservation-programs/filplus.storage#133]
Hello! As I understand it, the instructions on this site no longer take visitors to a place where they can get datacap. Is that right? Can we redirect them to a page that explains the current process, and/or a list of allocators?

**Comment by orvn** [comment]
Yes, we definitely can add something. I'm actually not too sure about what the future of this project will be at the moment, so it hasn't had many updates in the last couple of months.

**Comment by Kevin-FF-USA** [comment]
Thanks for kicking this off @dannyob 

Hi @orvn,
This would be great. But I image is tricky right now.
For redirect on the https://filplus.storage/apply would need to give a list of Allocators. That data lives in the JSON file in the Registry repo. Since there is no longer a single LDN, would need to direct applicants to each individual Allocator repo. **Whats the lift to pull and list this data on the filplus page that would list that?**

For context - here is where the data is

https://github.com/filecoin-project/Allocator-Registry/tree/main/Allocators

Imagine the list in https://filplus.storage/apply would need to have
  "name": "Public Open Dataset Pathway",
  "organization": "Data Preservation Institute",
  "location": "North America",
  "status": "Active",
  "metapathway_type": "Manual",
 "target_clients": [
      "Enterprise Data Clients",
      "Small-scale developers or data owners",
      "Individuals learning about Filecoin",
      "Other (specified above)"
      "required_sps": "2+",
    "required_replicas": "2+",
       "allocation_bookkeeping": "https://github.com/fidlabs/Open-Data-Pathway"
  },

**Comment by Kevin-FF-USA** [comment]
Hey @orvn 
If in the meantime - possible to make these (3) low lift changes?

1. Remove - Roadmap link
2. Remove - Apply Link
3. Add - Under Construction Banner

<img width="1348" alt="image" src="https://github.com/data-preservation-programs/filplus.storage/assets/94004586/cbb7b0b2-d142-479a-838f-bf201f675d33">

**Comment by smagdali** [comment]
We could create a page on docs explaining the new process and redirect the various old sites there?
